src/actions/getJwtToken.ts

```diff
@@ -2,12 +2,47 @@ import { functions } from '../firebase/firebase'
 import { UID } from '../../types/uid'
 import { JWTToken } from '../../types/JWT'
 
+const RETRY = {
+    number: 4,
+    interval: 300,
+}
+
+async function tryAtMost(promise, maxRetries, retryInterval) {
+    let retriesNumber = 0
+    while (retriesNumber <= maxRetries) {
+        try {
+            const result = await promise
+            return result
+        } catch (err) {
+            retriesNumber++
+            await new Promise((resolve) => {
+                setTimeout(function () {
+                    resolve(true)
+                }, retryInterval)
```

Comment:
We need to increase the delay, maybe more something like this: 
```
setTimeout(, retriesNumber^1.3)
```

Reply:
why ? Isn't retry with increased delay a retry practice not to make lots of calls, in case a service may be down or can't answer ? In our case we only need the hook to have been executed

Reply:
yes, the trigger may be cold (because it's a serverless) and the first time can take up to 2 or 3s (hot is aready used within last 10min =  much shorter delay). It's a common practice within mobile apps to increase the retry delay. + I don't know the exact delay of the login trigger.